modules/nw-ingress-configuring-application-domain.adoc

```diff
@@ -0,0 +1,75 @@
+// Module included in the following assemblies:
+//
+// * ingress/configure-ingress-operator.adoc
+//
+
+[id="nw-ingress-configuring-application-domain_{context}"]
+= Configuring application domain for the Ingress Controller Operator on AWS
+
+//OpenShift Dedicated or Amazon RH OpenShift cluster administrator
+
+As a cluster administrator, you can specify an alternative default domain for user-created routes by configuring `appsDomain` field. If you specify an alternative domain, it overrides the default cluster domain for the purpose of determining the default host for a newly created route.
+
+For example, you can use the DNS domain for your company as the default domain for routes and ingresses for applications running on your cluster.
+
+.Prerequisites
+
+//* You deployed an {OSD} cluster.
+* You deployed an {product-title} cluster on AWS infrastructure.
+* You installed the `oc` command line interface.
+
+.Procedure
+
+Configure the `appsDomain` field by specifying an alternative default domain for user-created routes.
+
+. Edit the Ingress Controller Operator:
++
+[source,terminal]
+----
+$ oc edit ingresses.config/cluster -o yaml
+----
++
+. Edit the YAML file:
++
+.Sample `appsDomain` configuration to `apps.acme.io`
+[source,yaml]
+----
+apiVersion: config.openshift.io/v1
+kind: Ingress
+metadata:
+  name: cluster
+spec:
+  domain: apps.<domain_url>
+  appsDomain: apps.acme.io
+----
++
+. Verify that the created route uses `appsDomain`:
++
+.. Access the Ingress Controller Operator YAML file:
++
+[source,terminal]
+----
+$ oc get ingresses.config/cluster -o yaml
+----
++
+[NOTE]
+====
+Wait for the `openshift-apiserver` finish rolling updates before exposing the route.
+====
++
+.. Expose the route:
```

Comment:
Which route? I feel like something's missing between looking at the Ingress Controller Operator YAML and exposing a route.